libsecret.go

```diff
@@ -137,6 +137,18 @@ func (k *secretsKeyring) Get(key string) (Item, error) {
 	return ret, err
 }
 
+// GetMetadata for libsecret returns an error indicating that it's unsupported
+// for this backend.
+//
+// libsecret actually implements a metadata system which we could use, "Secret
+// Attributes"; I found no indication in documentation of anything like an
+// automatically maintained last-modification timestamp, so to use this we'd
+// need to have a SetMetadata API too.  Which we're not yet doing, but feel
+// free to contribute patches.
+func (k *secretsKeyring) GetMetadata(key string) (Metadata, error) {
```

Comment:
I think at least the gnome-keyring implementation of DBus Secret Service might implement both a created and last-modified timestamps.

I've been working on some code that usies the keybase/go-keychain/secretservice package and I am able to create and then query and item and see both create and last-modified updated as expected:

```
# ./gnome-keyring-cli items
Default keyring: login
---
Type: GENERIC_SECRET
Name: testlabel
Secret: foo-18213
mtime: Wed May 29 00:35:24 2019
ctime: Tue May 28 13:43:02 2019
```

I'm not sure how to query that with the go-libsecret package used by this project, however.

Reply:
Ok, I think I see. The ctime/mtime looks like it's available from the `org.freedesktop.Secrets.Collection` interfaces `Items` property:

https://freedesktop.org/wiki/Specifications/secret-storage-spec/secrets-api-0.1.html#eggdbus-property-org.freedesktop.Secrets.Collection:Items